src/GafferSceneUIModule/SceneGadgetBinding.cpp

```diff
@@ -214,6 +214,7 @@ void GafferSceneUIModule::bindSceneGadget()
 		.def( "getSelection", &SceneGadget::getSelection, return_value_policy<copy_const_reference>() )
 		.def( "selectionBound", &selectionBound )
 		.def( "bound", &bound, ( arg( "selected" ), arg( "omitted" ) = object() ) )
+		.def( "snapshotToFile", &SceneGadget::snapshotToFile )
```

Comment:
Might be worth releasing the GIL for this, since it's a relatively heavy operation?